index.html

```diff
@@ -7,23 +7,49 @@
     <link rel="stylesheet" href="style.css" />
     <title>Fabien Portfolio</title>
   </head>
+
   <body>
     <div class="container">
       <header class="top">
         <a class="logo" href="#">Fabien</a>
         <nav class="ham">
-          <img class="hami" src="./img/Union.png" alt="hamburger img" />
+          <button class="button" type="button">
+            <span class="hams"></span>
+            <span class="hams"></span>
+            <span class="hams"></span>
+          </button>
+          <button class="clos" type="button">
+            <span class="cross"></span>
+            <span class="cross"></span>
+          </button>
         </nav>
         <nav class="nav-bar">
-          <a class="adj" href="">Portfolio</a>
-          <a class="adj" href="">About</a>
-          <a class="adj" href="">Contact</a>
-          <a href=""><img class="letter" src="./img/Union1.png" alt="mail image" /></a>
+          <a class="adj one" href="#work">Portfolio</a>
+          <a class="adj two" href="#fab">About</a>
+          <a class="adj three" href="#co">Contact</a>
```

Comment:
- [ ] when I hover over your navigation links the background becomes yellow and that doesn't match the Figma design so please try to remove it 
![image](https://user-images.githubusercontent.com/61790587/167680338-6f996cea-6638-4218-af7d-66f3da877deb.png)